db/revision.go

```diff
@@ -428,6 +428,7 @@ func containsUserSpecialProperties(b Body) bool {
 				BodyDeleted,
 				BodyAttachments,
 				BodyRevisions,
+				BodyExpiry,
```

Comment:
How come this got added? Was something failing?

Reply:
Yes TestDocExpiry was failing. We allow users to specify _exp in the body (as far as I understand?) but if I don't add this then users cannot do this and get the error `user defined top level properties beginning with '_' are not allowed in document body`

Reply:
I'm still unclear why `TestDocExpiry` used to work and now it doesn't without this fix.